pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: markdown
-version: 1.0.0-dev
+version: 1.0.1
```

Comment:
haven't released 1.0.0 yet. So leave it at dev

Reply:
why have we not released 1.0.0 yet?

Reply:
Because...it hasn't...haven't got around to it.

Reply:
Isn't it just take this PR and hit publish?